README.md

```diff
@@ -148,7 +148,7 @@ The URLs to download the data are constructed according to the following pattern
 - {format} is `netcdf`. Training output is also available as GRIB file,  using `format='grib'` and replacing `".nc"` by `".grib"`
 - {parameter} is `t2m` for [surface temperature at 2m](https://confluence.ecmwf.int/display/S2S/S2S+Surface+Air+Temperature), `tp` for [total precipitation](https://confluence.ecmwf.int/display/S2S/S2S+Total+Precipitation)
 - {origin} : `ecmwf` or `eccc` or `ncep` <!-- we should have a clean table for this once -->
-- {weeks} from [`"34"`, `"56"`, `["34", "56"]`] only for `benchmark` <!-- Todo @florian merge and use pd.Timedelta('14/28 d') -->
+- {weeks} from [`"34"`, `"56"`, `["34", "56"]`] only for `benchmark` <!-- Todo @florian merge and use pd.Timedelta('14/28 d') Do we still need this? -->
```

Comment:
```suggestion
```

Reply:
To stay consistent, if we rename 2t as t2m, we should also rename ci as siconc.

2t and ci are following the naming of the MARS database (internal to ECWMF),
t2m and siconc are following the CF conventions and they are the name of the dataarray provided by CliMetLab and in the netcdf.

Reply:
replied above. whats you opinion to deleting `weeks`?

Reply:
I don't mind deleting this. The data is there with both format anyways.

Reply:
ok. if both is there, we can also leave as is. I personally think it is better having a clean and consistent API.